doc/onboarding.md

```diff
@@ -0,0 +1,156 @@
+## onboarding to nodejs
+
+### **thank you** for doing this
+  * collaborators are effectively part owners
+  * going to cover four things:
+    * some project goals & values
+    * local setup
+    * issues, labels, and reviewing code
+    * merging code
+
+### a little deeper about the project
+  * effectively has the goals of it's contributors
+  * but, there are some higher-level goals and values
+    * not everything belongs in core (if it can be done reasonably in userland, let is stay in userland)
+    * empathy towards users matters (this is in part why we onboard people)
+
+### setup:
+  * notifications setup
+    * use https://github.com/notifications or set up email
+    * watching the main repo will flood your inbox, so be prepared
+  * CI testing:
+    * lives here: https://ci.nodejs.org/
+    * make sure to log in – I've sent your emails to rvagg, who will set up an account for you
+    * go to "Build with parameters"
+    * fill in user, repo, and branch. don't need to touch nodes_subset.
+  * git:
+    * make sure you have whitespace=fix: `git config --global --add core.whitespace fix`
+    * fork the repo to your user account on github
+    * `git clone git://github.com/USERNAME/node.git`
+    * `git remote add upstream git://github.com/nodejs/node.git`
+    * to update from nodejs/node:
+    * `git checkout master`
+    * `git remote update -p` OR `git fetch --all` (I prefer the former)
+    * `git merge --ff-only upstream/master` (or `REMOTENAME/BRANCH`)
+    * make new branches for all commits you make!
+  * #io.js on freenode is the best place to be to interact with the TSC / CTC / other collaborators
+    * ~~if you don't have IRC or a bouncer, contact me via email after the meeting and
+    we'll get you set up.~~
+
+### managing the issue tracker
+  * you have free reign – don't hesitate to close an issue if you are confident that it should be closed
+    * this will come more naturally over time
+    * IMPORTANT: be nice about closing issues, let people know why, and that issues and PRs can be reopened if necessary
+  * subsystems:
+    * we generally sort by these.
+    * lib/*.js
+    * doc, build, tools, test, deps, lib / src (special)
+  * cc'ing humans (by subsystem):
+    * lib/child_process: cjihrig, bnoordhuis, piscisaereus
+    * lib/cluster: cjihrig, bnoordhuis, piscisaereus
+    * lib/stream*: nodejs/streams
+    * lib/net: indutny, bnoordhuis, piscisaereus, chrisdickinson, nodejs/streams
+    * lib/http: indutny, bnoordhuis, nodejs/http
+    * lib/vm: domenic
+    * lib/buffer: trevnorris
+    * src/async-wrap.*: trevnorris
+    * lib/crypto,tls,https: indutny, shigeki
+    * src/node_crypto.*: indutny, shigeki
+    * lib/timers: fishrock123
+    * upgrading v8: bnoordhuis / targos / ofrobots
```

Comment:
hmm... I've been thinking more and more that these should be teams like nodejs/http as opposed to specific individuals. For one, it makes it easier to remember and easier to mention multiple people.